docs/tutorial/rust/src/getting_started.md

```diff
@@ -18,7 +20,7 @@ Then we edit `Cargo.toml` to add the slint dependency:
 edition = "2021"
 
 [dependencies]
-slint = "0.2.0"
+slint = "0.2.1"
```

Comment:
Suggestion from Olivier which makes it even clearer that we're happy with any `0.2.x` release:

```suggestion
slint = "0.2"
```

Reply:
I think this is going to be anyway overwritten by the script that updates the version number in the docs